docs/1manifoldLearning.md

```diff
@@ -0,0 +1,48 @@
+# 1-Manifold Learning example
+
+![1-Manifold Learning example Image](https://topology-tool-kit.github.io/img/gallery/1manifoldLearning.jpeg)
+
+
+## Pipeline description
+
+This example first loads a point cloud from disk. 
+
+In a pre-processing, the [DimensionReduction](https://topology-tool-kit.github.io/doc/html/classttk_1_1DimensionReduction.html) is used to reduce the dimension of the input to 3D points. The data is then converted to a format understandable by Paraview using the TableToPoints filter. Gaussian Resampling is applied to the data (upper left view in the above screenshot). This filter has the effect of injecting input points to a structured data. For each injection, each point will "splat", or distribute values to nearby vertices. The resulting SplatterValues field is density estimation (with a Gaussian kernel) of the point cloud projected in 2D.
+
+Then, the [PersistenceDiagram](https://topology-tool-kit.github.io/doc/html/classttkPersistenceDiagram.html) of a slice of the obtained data is computed and thresholds are applied based on persistence to maintain only the most persistent features. This results in a simplified persistence diagram.
+
+The simplified persistence diagram is then used as a constraint for the [TopologicalSimplification](https://topology-tool-kit.github.io/doc/html/classttkTopologicalSimplification.html) of the input scalar data.
+
+This simplified data is then used as the input of the computation of [MorseSmaleComplex](https://topology-tool-kit.github.io/doc/html/classttk_1_1MorseSmaleComplex.html) (right view, above screenshot). This complex is composed of elements of 3 dimensions: dimension 0, which corresponds to the critical points of the Morse-Smale Complex, dimension 1, which corresponds to its edges (in grey in the screenshot) and dimension 2, which corresponds to its surfaces. Only certain edges are displayed here: using thresholds, the edges connecting at least one critical point situated in the boundary are discarded. This way, the "S" shape made by the point cloud is outlined.
```

Comment:
"(in grey in the screenshot)" -› ""
(I don't believe they appear in this screenshot)

Reply:
"Only certain edges are displayed here" -› "Only certain maximal edges are displayed here"
(in fact, the first threshold selects the separatrices connected to maxima)